### PR TITLE
WIP Auth -  Made basic login functionality

### PR DESCRIPTION
**Pull request by abhij1607**
image

- added static wishlist UI page
- add login UI page
- added logIn function

### CODE REVIEW

File: src/pages/Login/login.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [response, setResponse] = useState("");
+
+  const loginRequest = async (e) => {
+    e.preventDefault();
+    const data = { email, password };
+    console.log(data);
```
Comment (junaidshaikh-js) [inline src/pages/Login/login.js]: console log should be removed from the code

File: src/pages/Login/login.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    try {
+      const loginResponse = await axios.post("/api/auth/login", data);
+      console.log(loginResponse);
+      setResponse(loginResponse.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {});
```
Comment (junaidshaikh-js) [inline src/pages/Login/login.js]: Empty useEffect for what?

File: src/pages/Login/login.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                value={password}
+                onChange={(e) => setPassword(e.target.value)}
+              />
+            </div>
+            <div className="input-checkbox">
+              <label>
+                <input type="checkbox" />
+                Remember me
+              </label>
+              <a href className="align-right">
```
Comment (junaidshaikh-js) [inline src/pages/Login/login.js]: href attribute should have some value

**Review by junaidshaikh-js — COMMENTED** [review]
Good work! 

I mentioned some points in the comments.

Other than that everything looks perfectly fine. After making required changes the PR can me merged.

**Review by prachi-sahani — COMMENTED** [review]
Apart from the points mentioned by Junaid, code is well structured and good to merge

**Comment by abhij1607** [comment]
made the pointed out changes, now merging PR

**Pull request merged**